public/app/features/connections/tabs/ConnectData/CardGrid/CardGrid.tsx

```diff
@@ -0,0 +1,74 @@
+import { css } from '@emotion/css';
+import React, { FC } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Card, LoadingPlaceholder, useStyles2 } from '@grafana/ui';
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  sourcesList: css`
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
+    gap: 12px;
+    list-style: none;
+    margin-bottom: 80px;
+  `,
+  card: css`
+    height: 90px;
+    padding: 0px 24px;
+    box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.6);
+  `,
+  cardContent: css`
+    display: flex;
+    align-items: center;
+  `,
+  logoWrapper: css`
+    display: flex;
+    justify-content: center;
+    margin-right: 8px;
+    width: 32px;
+    height: 32px;
+    img {
+      max-width: 100%;
+      max-height: 100%;
+      align-self: center;
+    }
+  `,
+  label: css`
+    color: ${theme.colors.text.primary};
+    margin-bottom: 0;
+  `,
+});
+
+export interface CardGridProps {
+  items: Array<{ id: string; name: string; logo?: string }>;
+  isLoading: boolean;
```

Comment:
Would it make sense to make the `isLoading` prop optional so in case it's not passed in then the default state is just "not loading"?